examples/menubar/menubar-2/menubar-2.html

```diff
@@ -60,18 +60,22 @@ <h2 id="ex1_label">Example</h2>
         <li role="none">
           <a role="menuitem" aria-haspopup="true" aria-expanded="false" href="#">Style/Color</a>
           <ul role="menu" aria-label="Style/Color">
-            <li role="menuitemcheckbox" data-option="font-bold" aria-checked="false">Bold</li>
-            <li role="menuitemcheckbox" data-option="font-italic" aria-checked="false">Italic</li>
+            <li role="none">
+              <ul role="group" rel="font-style" aria-label="Font Style">
```

Comment:
I don't think the `rel` value is correct. I think that was striped off to the data attributes before
```suggestion
              <ul role="group" data-option="font-style" aria-label="Font Style">
```